src/pane.coffee

```diff
@@ -336,6 +336,8 @@ class Pane extends Model
   #
   # Returns the added item.
   addItem: (item, index=@getActiveItemIndex() + 1) ->
+    throw new Error("Added a null item") unless item? and typeof item is 'object'
```

Comment:
You're right. Oversight. I expanded to non-objects and forgot to update the message.